src/SonarQube.Client/Api/Requests/V6_30/GetPropertiesRequest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using SonarQube.Client.Models;
+
+namespace SonarQube.Client.Api.Requests.V6_30
+{
+    public class GetPropertiesRequest : RequestBase<SonarQubeProperty[]>, IGetPropertiesRequest
+    {
+        protected override string Path => "api/settings/values";
+
+        protected override SonarQubeProperty[] ParseResponse(string response) =>
+            JObject.Parse(response)["settings"]
+                .ToObject<PropertyResponse[]>()
+                .Select(ToProperty)
+                .ToArray();
+
+        private SonarQubeProperty ToProperty(PropertyResponse arg) =>
+            new SonarQubeProperty(arg.Key, arg.Value);
+
+        private class PropertyResponse
+        {
+            [JsonProperty("key")]
+            public string Key { get; set; }
+
+            [JsonProperty("value")]
+            public string Value { get; set; }
+
+            // TODO: fieldValues
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)